README.md

```diff
@@ -46,17 +46,7 @@ Copy the token in the format of `workspace:token` and pass it to the `gpud up --
 sudo gpud up --token <LEPTON_AI_WORKSPACE:TOKEM>
 ```
 
-To check the status of the running gpud:
-
-```bash
-sudo gpud status
-```
-
-To check the logs of the running gpud:
-
-```bash
-sudo gpud logs
-```
+You can go to the [dashboard](https://dashboard.lepton.ai) to check the self-managed machine status.
```

Comment:
can we directly link to the machine/self-managed section?

Reply:
@bobmayuze

Reply:
Not yet. Will update when we support it.

Reply:
Let me update